src/constants/chainInfo.ts

```diff
@@ -46,7 +46,7 @@ export type ChainInfoMap = { readonly [chainId: number]: L1ChainInfo | L2ChainIn
   readonly [chainId in SupportedL2ChainId]: L2ChainInfo
 } & { readonly [chainId in SupportedL1ChainId]: L1ChainInfo }
 
-const CHAIN_INFO: ChainInfoMap = {
+export const CHAIN_INFO: ChainInfoMap = {
```

Comment:
we shouldn't export chain_info. but there's a function below called `getChainInfo`. can you add a comment that we shouldn't export this map?